css/fonts.css

```diff
@@ -47,26 +48,25 @@ h5{
 
 
 
-
 p{
 	font-size: 1.1em;
 	margin: 10px;
 	color: black;
+	font-family: arial;
 }
 
 
 
+
 p1{
 	color: white;
+	font-family: arial;
 }
 
 
-
-.grey{
```

Comment:
took code out as I wasn't using this class anymore; from teacher practice database